sources/Valkey.Glide/Commands/Options/HashSetExOptions.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+namespace Valkey.Glide.Commands.Options;
+
+/// <summary>
+/// Optional arguments for the HSETEX command.
+/// </summary>
+/// <remarks>
+/// See <a href="https://valkey.io/commands/hsetex/">valkey.io</a>
+/// </remarks>
+public class HashSetExOptions
+{
+    /// <summary>
+    /// Expiration settings for the hash fields.
+    /// </summary>
+    public ExpirySet? Expiry { get; set; }
+
+    /// <summary>
+    /// When true, only set fields if none of them exist.
```

Comment:
nit: For me it is difficult to track if this is the FNX | FXX. It would be good if the acronym be in the comment as well. This apply for all the other comments, but this it is your call.

Reply:
We have not mapped option sets to abbreviated commandline arguments anywhere else in documentation. Client libraries intentionally hide the implementation details of the underlying protocols as an individual API call may involve multiple Valkey/Redis commands.